.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,14 @@
+### Milestone (Odoo version)
```

Comment:
I don't, just a possibility.
User can remove the template if it's not relevant. But in 90% of the cases, this a new module or a feature or a fix, so, milestone and module name are relevant. Don't you think ?